docs/root/configuration/cluster_manager/cluster_stats.rst

```diff
@@ -10,7 +10,9 @@ General
 -------
 
 The cluster manager has a statistics tree rooted at *cluster_manager.* with the following
-statistics. Any ``:`` character in the stats name is replaced with ``_``.
+statistics. Any ``:`` character in the stats name is replaced with ``_``. All stats concerning
+number of clusters (other than cluster_modified and _removed) include statically specified
+xDS servers in the count.
```

Comment:
This is not strictly true for Google gRPC xDS servers; I'd leave this one out or rephrase.

Reply:
Ah, ok - if it's that complicated, maybe a vague warning is best, so that people just know to be on the lookout, and to investigate how it will work in their specific circumstances. I could also just leave it out, but the fact that xDS servers themselves are counted was a surprise to me, so it might be useful.

Reply:
Well, with Envoy gRPC, the xDS server is a cluster, like any other other cluster. Why would it be surprising that they are included in cluster stats given this data model?

Reply:
Most of this page sounds like it's talking about the actual regular upstreams, for instance the load balancing sections. So, I would have expected that these stats are all about the work Envoy is doing, rather than how it is doing it.

If it always behaved one way, I could see leaving the extra detail out, and accepting that this page is a reference for people who already know how the system works, rather than being intended to inform beginners. However, there is some real subtlety here to be documented. I think it certainly is surprising that choosing a different implementation of gRPC results in different values of stats (other than stats directly about gRPC performance).